en/rental-space-de-activation.md

```diff
@@ -0,0 +1,25 @@
+# Rental Space De/Activation
+
+This is a quick guide to show you how to `Deactivate` and `Activate` back your rental spaces.
+
+📌 Inactive spaces are not available for new bookings, however, their historical bookings are not canceled automatically
+
+## Deactivation
+
+1. Go to the Rental Spaces section where you can find all your created rental spaces. Select the `Active` section in the list.
+
+2. Select the spaces you want to deactivate and click the `Deactivate` button which is right next to the select all button.
+
+   ![Alt text](../img/Screenshot%202022-08-30%20at%2014.31.13.png?raw=true "Sakurabook Rental Space Deactivation")
+
+## Activation
+
+1. Go to the Rental Spaces section where you can find all your created rental spaces. Select the `Inactive` section in the list.
+
+2. Select the spaces you want to deactivate and click the `Activate` button which is right next to the select all button.
+
+   ![Alt text](../img/Screenshot%202022-08-30%20at%2014.31.24.png?raw=true "Sakurabook Rental Space Deactivation")
+
```

Comment:
I would add info, that the space will be still visible to the customer and if they want to hide it, they need to make it draft in Shopify admin